src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -248,6 +248,16 @@ export const createRequestSignature = (
     };
 };
 
+/**
+ * Generate a schnorr signature using the group's private key over the group's provided id
+ * @param groupPrivateKey
+ * @param groupPublicKey
+ * @param groupId
+ */
+export const generateAddMemberSignature = (groupPrivateKey: Uint8Array, groupPublicKey: PublicKey<string>, groupId: string) => {
+    return Future.of(RecryptApi.schnorrSign(groupPrivateKey, publicKeyToBytes(groupPublicKey), utf8StringToArrayBuffer(groupId)));
```

Comment:
I'm not sure there's a reason for this to return a Future if it doesn't need to. Similar to the `createRequestSignature` above since there's no async calls happening I'd probably just directly return the `schnorrSign` call and wrap call so this function in a Future where needed.

Reply:
done